### ISSUE TITLE
How to select cfg file in training? so many cfg files.

### ISSUE THREAD

**Issue by DanLiu0623** [AlexeyAB/darknet#300]
I use yolo-voc.cfg  or tiny-yolo-voc.cfg generally due to my training data is voc format, But there are other cfg files, how should I choose ?

**Comment by AlexeyAB** [comment]
1. These cfg files are for the detection - to say what objects and where are objects on the image:
    `yolo-voc.cfg, yolo-voc.2.0.cfg, tiny-yolo-voc.cfg, yolo.cfg, yolo.2.0.cfg, tiny-yolo.cfg` - they differ speed and accuracy: https://pjreddie.com/darknet/yolo/

2. These cfg files are for the detection - to say what objects on the image:
    `alexnet.cfg, cifar.cfg, darknet.cfg, vgg-16.cfg, darknet19.cfg, darknet448.cfg, resnet50.cfg, densenet201.cfg` - they differ speed and accuracy: https://pjreddie.com/darknet/imagenet/

3. There are recurrent netowkrs: `rnn.cfg, rnn.train.cfg` - to generate text in the style of great writers: https://pjreddie.com/darknet/rnns-in-darknet/

**Comment by stephanecharette** [comment]
You are replying to a very old message.  I have a page that lists the various configuration files, and some information on each one: https://www.ccoderun.ca/darkmark/Configuration.html

And yes, you should take the one that matches what you'd like to do, and then modify it accordingly if you want to train your own custom neural network.
